src/Infrastructure/LeanCode.ConfigCat/ConfigCatInitializer.cs

```diff
@@ -23,6 +23,9 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         {
             await configCatClient.GetAllKeysAsync(stoppingToken);
         }
-        catch { }
+        catch
+        {
+            // We don't want any exceptions to be propagated.
```

Comment:
Okay, but _why_? :P

Reply:
It was just copied from the `SupressMessage` above in order to satisfy Rider. I adjusted the comment to be more explanatory but I will maybe ask @Saancreed if this is what he meant here.

Reply:
He does not remember but I saw on config cat PR that it logs message by itself so no need to log it here. I added it to comment.